modules/stanford_news_views/stanford_news_views.views_default.inc

```diff
@@ -516,20 +517,48 @@ function stanford_news_views_views_default_views() {
   $handler->display->display_options['empty']['area']['field'] = 'area';
   $handler->display->display_options['empty']['area']['label'] = 'News Item - No Results';
   $handler->display->display_options['empty']['area']['empty'] = TRUE;
-  $handler->display->display_options['empty']['area']['content'] = 'No news items listed.';
+  $handler->display->display_options['empty']['area']['content'] = '<p>See all announcements and news in the archive.</p>
+<p><a class="more-link" href="news/announcements">View past announcements</a></p>';
```

Comment:
@cynmij,

This URL will not work. You cannot link to a relative path in this content. It will break. 

Can you create help text that does not have a link in it?